src/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -5,12 +5,80 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
+using System.Runtime.CompilerServices;
 using Xunit;
 
 namespace System.Tests
 {
     public partial class DoubleTests : RemoteExecutorTestBase
     {
+        // NOTE: Consider duplicating any tests added here in SingleTests.cs
+
+        private static ulong DoubleToUInt64Bits(double value)
```

Comment:
Why not use [https://docs.microsoft.com/en-us/dotnet/api/system.bitconverter.doubletoint64bits?view=netframework-4.7.2](https://docs.microsoft.com/en-us/dotnet/api/system.bitconverter.doubletoint64bits?view=netframework-4.7.2) instead?

Reply:
Because that converts to `long` and makes checks against hex values harder. That is, you can no longer do: 
```
Assert.Equal(0xFFF00000_00000000u, DoubleToUInt64Bits(double.NegativeInfinity));
```
you have to do:
```
Assert.Equal(unchecked((long)(0xFFF00000_00000000u)), BitConverter.DoubleToInt64Bits(double.NegativeInfinity));
```

Reply:
Right, will fix.

I had just modified the code I put into `SingleTests`, since `BitConverter.SingleToInt32Bits` isn't available on Desktop.

Reply:
Fixed.